modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -149,31 +169,117 @@ function inope_policy_review_form_submit($form, &$form_state) {
-    drupal_set_message(
-      t('User : @userid : @username submitted the following change request:', array(
-        '@userid' => $account->uid,
-        '@username' => $account->name,
-      )));
 
-    drupal_set_message($form_state['values']['inope_policy_review_changes_requested_textarea']);
+    $policy_review_id = inope_policy_review_insert_query($node, $account, $text, 0);
 
-    drupal_set_message(t('Node ID : @nodeid', array('@nodeid' => $node->nid)));
-    drupal_set_message(t('Node VID : @nodevid', array('@nodevid' => $node->vid)));
-    // TODO: put this into the database.
+    if ($policy_review_id != 0) {
+      // Thank the user for their request.
+      drupal_set_message(
+        t('Thank you @username for submitting the following change request:', array(
+          '@username' => $account->name,
+        )));
+      // Check_plain happened when this variable was declared.
+      drupal_set_message($text);
+    }
   }
   // If a bug is encountered.
   else {
     drupal_set_message(t('Form state not recognized'));
   }
+}
+
+/**
+ * Select query for the current node policy review notes.
+ *
+ * Query the policy review table for any records related to the current
+ * node, note status, and are from the current user.
+ */
+function inope_policy_review_select_query($node, $account) {
+
+  // Select query to get the latest node revision timestamp.
+  $node_revision = db_query("
+  SELECT vid, timestamp FROM {node_revision} WHERE vid = :vid", array(
+    ':vid' => $node->vid,
+  ))->fetchObject();
+
+  // Select query to get all records with a note_sumbitted >= vid_timestamp.
+  $policy_review_notes = db_query("
+  SELECT * FROM {policy_review} WHERE nid = :nid AND uid = :uid AND 
+  note_submitted >= :vid_timestamp ORDER BY note_submitted DESC", array(
+    ':nid' => $node->nid,
+    ':uid' => $account->uid,
+    ':vid_timestamp' => $node_revision->timestamp,
+  ))->fetchAll();
+
+  return $policy_review_notes;
+}
+
+/**
+ * Insert query to add a new policy review note or approve policy.
+ *
+ * Only insert the new row if it is substantially a new request meaning that
+ * some value other than the timestamp is different.
+ */
+function inope_policy_review_insert_query($node, $account, $text, $status) {
+
+  $results = inope_policy_review_select_query($node, $account);
+
+  // If the entry already exists don't duplicate it.
+  if (!empty($results)) {
+    drupal_set_message(t('You have previously submitted a response.'));
+    $policy_review_id = 0;
+  }
+  else {
+    // Run an insert query on the policy review table for the new note.
+    $policy_review_id = db_insert('policy_review')
+      ->fields(array(
+        'nid' => $node->nid,
+        'vid' => $node->vid,
+        'uid' => $account->uid,
+        'note_status' => $status,
+        'note_submitted' => time(),
+        'note' => $text,
+      ))
+      ->execute();
+  }
+
+  return $policy_review_id;
+}
+
+/**
+ * Update query if the user wants to update a record.
```

Comment:
Ah I think there was yet another misunderstanding -- that if we had had more time we could have cleared up. It isn't important that a staff member always be able to update a response they had but rather -- they should be able to update only their current response. I now see why you chose an update here.